config/jetty.yml

```diff
@@ -1,10 +1,10 @@
 development:
   startup_wait: 15
-  jetty_port: 8983
+  jetty_port: 8980
```

Comment:
Probably, yeah. Do we want to do that as part of this PR? Switching the port happened here so that Plum and Pom could run together in dev.